src/server.js

```diff
@@ -41,7 +41,7 @@ var walk = function(path) {
 walk(models_path);
 
 // bootstrap passport config
-require('./config/passport')(passport);
+require('./config/passport').default(passport);
```

Comment:
use imports at the top and call fn here
e.g. `import expressConfig from './config/express'
expressConfig(app, passport, mongoose)`

same for others, to avoid using require and default